Addition info for Okta SCIM (#7896)
fishtown-analytics/docs.getdbt.com: website/docs/docs/cloud/manage-access/scim.md

You've now configured SCIM for the Okta SSO integration in <Constant name="cloud" />.

### SCIM username format

> SCIM requires the username format to be set to the user's email address. If your Okta configurations map the `userName` attribute to a different format, SCIM user provisioning will fail. To set your Okta app config to `email`:

Author: ```suggestion
SCIM requires the username to be in the email address format. If your Okta configurations map the `Username` field to a different attribute, SCIM user provisioning will fail. To get around this without altering your user profiles, set your Okta app config to `email`:
```

> SCIM requires the username to be in the email address format. If your Okta configurations map the `Username` field to a different attribute, SCIM user provisioning will fail. To get around this without altering your user profiles, set your Okta app config to `email`:

Reviewer: Line 99 uses `Email`. If the UI uses `Email` in the dropdown menu, I suggest we also use the same case in the description.
```suggestion
SCIM requires the username to be in the email address format. If your Okta configurations map the `Username` field to a different attribute, SCIM user provisioning will fail. To get around this without altering your user profiles, set your Okta app config to `Email`:
```

Author: Good catch!

1. Open the SAML app created for the dbt integration.

> 2. In the apps **Sign on** tab, click **Edit** in the **Settings** pane.

Author: ```suggestion
2. In the **Sign on** tab, click **Edit** in the **Settings** pane.
```
